config/provider_config.go

```diff
@@ -75,14 +76,28 @@ func (c *ProviderConfig) Init(rc *RootConfig) error {
 	if c == nil {
 		return nil
 	}
+	logger.Debug("test")
+	buildDebugMsg := func() string {
```

Comment:
buildDebugMsg := func() string {
		if len(c.Services) != 0 {
			providerNames := make([]string, 0, len(c.Services))
			for k := range c.Services {
				providerNames = append(providerNames, k)
			}
			return strings.Join(providerNames, ", ")
		} 
		return "empty"
	}

Reply:
OK, I have modified it.